_pages/saml.md

```diff
@@ -55,86 +55,124 @@ Here are values needed to configure your service provider (SP) to work with logi
+Z2luLmdvdjEaMBgGA1UEAwwRc3RhZ2luZy5sb2dpbi5nb3YwHhcNMjEwMzA1MTkw
+NzQ3WhcNMjIwNDAxMTkwNzQ3WjB/MQswCQYDVQQGEwJVUzEdMBsGA1UECAwURGlz
+dHJpY3Qgb2YgQ29sdW1iaWExEzARBgNVBAcMCldhc2hpbmd0b24xDDAKBgNVBAoM
+A0dTQTESMBAGA1UECwwJTG9naW4uZ292MRowGAYDVQQDDBFzdGFnaW5nLmxvZ2lu
+LmdvdjCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBANUNfCMWGoT9v4N/
+7I8q/bILj7uPERI0ECBooaY7mCAQijbwEJd0EvGnKDSYumqX0TNlcOaMk7cw0t1P
+gBTRgpIcRkUxHrUqDrEz2HIfDT6FMQuLoaq48HwEzeYLN51V9y1F2zqDCuQ5tP4z
+Vu1IrMeUDVhIzqwhDXpPit847lqcJjGulq3sIqOCLzncl28J8FST8jhNEUZZq7Kz
+LXjbofAtTOuCTnHKFeOD2Z54ltti3kzknodO3EMzSybTW6ZVrqkTwylwuMTcLIFA
++7MmrEtLZZA0+TMgeGzK2eHjDGyNVsLVsqNP4nWEzH+cYui2PkhPbF+H8FwzfNeT
+rfTttEcCAwEAAaNTMFEwHQYDVR0OBBYEFERlUaCo4VsdtmbGqkBBRxfOeFZkMB8G
+A1UdIwQYMBaAFERlUaCo4VsdtmbGqkBBRxfOeFZkMA8GA1UdEwEB/wQFMAMBAf8w
+DQYJKoZIhvcNAQELBQADggEBAJa2Pdn+gyMWiDOSp6D1IQPrHkIfkMxQNZ1VB/oe
+HXOWWiuxu1OFnG7Jn5MN+aQVWkUasBIfG8pMEsh0ymF7XvwvjMaflxSEb0ubgivg
+4+AUPwLKeTapr+aVozqxuRSjj7rJBKh6/u9Adip87rH92K/L/kBfky2gCJ6DAJYr
+nBO2k9yV7Wpil1OBK5DQ71Cvg1ZxQon1zvd7Iy0qUwbdXV48RPDk296+jGFYkxte
+kiLkLeRXCut/8L3YuI/zUeVVnb0Qn6OIC3ouVPingl6g9f15KHYoXwi2MpC6ZsbG
+zj6mqTIhwdCVxnZ8sm4HKtFKfCtyI3db3DVg/68hW+NZr9w=
 -----END CERTIFICATE-----
 ```
+  </div>
 </div>
+
+<div class="usa-accordion--bordered">
+  <button class="usa-accordion__button" aria-controls="prod-cert-2021">
+  View 2021 <strong>production</strong> certificate
+  </button>
+  <div id="prod-cert-2021" class="usa-accordion__content" markdown="1">
+```
+-----BEGIN CERTIFICATE-----
+MIIDzzCCAregAwIBAgIUZD8catEsxhppdBq7wJDo6qmr54kwDQYJKoZIhvcNAQEL
+BQAwdzELMAkGA1UEBhMCVVMxHTAbBgNVBAgMFERpc3RyaWN0IG9mIENvbHVtYmlh
+MRMwEQYDVQQHDApXYXNoaW5ndG9uMQwwCgYDVQQKDANHU0ExEjAQBgNVBAsMCUxv
+Z2luLmdvdjESMBAGA1UEAwwJbG9naW4uZ292MB4XDTIxMDMwNTE5MDc1OFoXDTIy
+MDQwMTE5MDc1OFowdzELMAkGA1UEBhMCVVMxHTAbBgNVBAgMFERpc3RyaWN0IG9m
+IENvbHVtYmlhMRMwEQYDVQQHDApXYXNoaW5ndG9uMQwwCgYDVQQKDANHU0ExEjAQ
+BgNVBAsMCUxvZ2luLmdvdjESMBAGA1UEAwwJbG9naW4uZ292MIIBIjANBgkqhkiG
+9w0BAQEFAAOCAQ8AMIIBCgKCAQEAxOOpDwiXOunf2brWjkaTpAV+pg60T6c6Fsyt
+EebNrrUMFPxXIP6eDbhSCiiM83koLSJek4bStma4me6RM7RM0+qLq526e08DE0lu
+DM0u45YsROWDqiAHFxJxqYovqYQ9JnaxuXmVuCfgBJyLQA0RcbN1G+NhT7VpgMzw
+ouyXhZs5O98fIGMBm7HVckxsude75yOLYOOhiTL2RQnYw8fjsz/B/kDZB+jcDFuR
+/y7ysVpVJ3/wt5lzuI10S1rLlr4XM09l1K/TgMXPJr4jbjm4SbmozhE4vC9UE0yu
+Wv6ZrXW+5gidhCOuy0BvMDZGXsqPJ0TIsQz1E/tSDz9gyke2nwIDAQABo1MwUTAd
+BgNVHQ4EFgQUIoi+9o/vRGxzAvObLqhB66KPY4kwHwYDVR0jBBgwFoAUIoi+9o/v
+RGxzAvObLqhB66KPY4kwDwYDVR0TAQH/BAUwAwEB/zANBgkqhkiG9w0BAQsFAAOC
+AQEACy3VqftGA5bVYMwcvBQFGCMwRqTyL8jGKnvt8RZJ8ePX2thgXIsaC5zPJQAx
+s4z7DFubSBZEpoBwxEIZVI8hsFkPKrUYgsBRTUBVA2Q3E48VTVxyVLEul8BESqaZ
+vIeYwOGwc6q14kLHmq/ZBzbeZPEH2wPORwXouWOonxAWAHvrUUjt0uEY3y0VXRD5
+LnmMjyTlRLqBQyxJWEnJYeeWhIk0ARJMcerkaLcMASUlL6Sua7aI8E2qXOImJofC
+I+mn1PzJcSXvJE/Xi3UVt0GMVgIK7ZsKnAnDR6JulJOq1lkLh3QvERfXQJ4Zz88V
+lzdI/A8lml7s8FEL3jF/rd6lUg==
+-----END CERTIFICATE-----
+```
+  </div>
 </div>
 
 #### Annual Certificate Rotation
```

Comment:
**Question:** Do we want to update this section as well to reference the 2021-2022 rotation?

Reply:
Yes! Updated in e335cab. Thanks!